Gopkg.toml

```diff
@@ -48,3 +48,11 @@
 [prune]
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  name = "google.golang.org/grpc"
+  version = "1.15.0"
+
+[[constraint]]
+  branch = "master"
```

Comment:
The code in branch `master` may change overtime. So I think it is better to keep either version or commit id.

Reply:
WIP, I will incorporate in future commits.